dev/index.ts

```diff
@@ -99,6 +101,19 @@ const editor2 = new EditorView({
   parent: document.querySelector("#editor-json5")!,
 });
 
+/**
+ * yaml!
+ */
+const yamlState = EditorState.create({
+  doc: yamlText,
+  extensions: [commonExtensions, yamlSchema(schema)],
+});
+
+const editor3 = new EditorView({
+  state: yamlState,
+  parent: document.querySelector("#editor-yaml")!,
+});
+
 const handleSchemaChange = (newSchema: JSONSchema7) => {
   updateSchema(editor1, newSchema);
   updateSchema(editor2, newSchema);
```

Comment:
ah! a useful review find finally haha. so you just need to add `updateSchema` with `editor3` here. for example, switching to github actions should show errors for the entire yaml document as well